mm2src/docker_tests.rs

```diff
@@ -1966,49 +1977,46 @@ mod docker_tests {
                 "rel": "MYCOIN1",
-                "swap_method": "sell",
+                "swap_method": "setprice",
                 "price": 1,
-                "volume": 0.01,
+                "max": true,
             },
         })))
         .unwrap();
         assert!(!rc.0.is_success(), "trade_preimage success, but should fail: {}", rc.1);
-        let available = BigDecimal::from(0);
-        // `required = volume + fee_to_send_taker_payment + dex_fee + fee_to_send_dex_fee`,
-        // where `volume = 0.01`, `fee_to_send_taker_payment = fee_to_send_dex_fee = 0.00001`, `dex_fee = 0.0001`.
-        // Please note `dex_fee = 0.01 / 7770` < `min_dex_fee = 0.0001`, so `dex_fee = min_dex_fee = 0.0001`
-        let required = MmNumber::from("0.01") + MmNumber::from("0.00012");
-        expect_not_sufficient_balance(&rc.1, available, required.to_decimal());
-
-        // The max available value = balance (0.000015) - transaction_fee (0.00001) = 0.000005 that is less than dust (0.00001).
-        // In this case we have to receive the `NotSufficientBalance` error.
-        //
-        // vvv Fill the MYCOIN balance vvv
-
-        let low_balance = MmNumber::from("0.000015").to_decimal();
-        let (_ctx, mycoin) = utxo_coin_from_privkey("MYCOIN", priv_key.as_ref());
-        let my_address = mycoin.my_address().expect("!my_address");
-        fill_address(&mycoin, &my_address, low_balance.clone(), 30);
+        // balance(0.000015)
+        let available = MmNumber::from("0.000015").to_decimal();
+        // balance(0.000015) + transaction_fee(0.00001)
+        let required = MmNumber::from("0.00002").to_decimal();
+        expect_not_sufficient_balance(&rc.1, available, required, None);
 
+        fill_balance_functor(MmNumber::from("0.099995").to_decimal());
         let rc = block_on(mm.rpc(json!({
             "userpass": mm.userpass,
             "mmrpc": "2.0",
             "method": "trade_preimage",
             "params": {
                 "base": "MYCOIN",
                 "rel": "MYCOIN1",
-                "swap_method": "setprice",
+                "swap_method": "sell",
                 "price": 1,
-                "max": true,
+                "volume": 0.1,
             },
         })))
         .unwrap();
         assert!(!rc.0.is_success(), "trade_preimage success, but should fail: {}", rc.1);
-        // balance(0.000015)
-        let available = low_balance;
-        // balance(0.000015) + transaction_fee(0.00001)
-        let required = MmNumber::from("0.00002").to_decimal();
-        expect_not_sufficient_balance(&rc.1, available, required);
+        let available = MmNumber::from("0.10001").to_decimal();
+        // `required = volume + fee_to_send_taker_payment + dex_fee + fee_to_send_dex_fee`,
+        // where `volume = 0.1`, `fee_to_send_taker_payment = fee_to_send_dex_fee = 0.00001`, `dex_fee = 0.0001`.
+        // Please note `dex_fee = 0.01 / 7770` < `min_dex_fee = 0.0001`, so `dex_fee = min_dex_fee = 0.0001`
```

Comment:
Please change `dex_fee = 0.01 / 7770` to `dex_fee = 0.1 / 7770` regarding new volume 0.1